### PR TITLE
Modal Fixes

### PR DESCRIPTION
**Pull request by al5ina5**
Temporary test fix for the metamask android bug.

Will stylize a bit better upon approval of working from Android device.

### CODE REVIEW

File: src/components/WalletProviderModal/WalletCard.js [hunk 1]
```diff
@@ -2,10 +2,21 @@ import React from 'react';
 import { ListItem, ListItemIcon, ListItemText } from '@material-ui/core';
 
 const WalletCard = ({ icon, onConnect, title }) => (
-  <ListItem button onClick={onConnect}>
-    <ListItemIcon>{icon}</ListItemIcon>
-    <ListItemText primary={title} />
-  </ListItem>
+  <button
```
Comment (OGBoogieTomb) [inline src/components/WalletProviderModal/WalletCard.js:5]: The overall styling of the popup seems to be changed. Styling changes to bring back:
- color of the text
- font of the text
- hover text color ( extra credibility if you set it to [this](https://github.com/tombfinance/tombfinance-frontend/blob/main/src/newTheme.js#L15))
- hover background color

**Review by OGBoogieTomb — CHANGES_REQUESTED** [review]
Hi, first of all thanks a ton for taking the time to look into the issue. Its very much appreaciated. 🎉 

Functionality is working as expected. 
A couple of things that need to be fixed before this is merged.

Left a comment.

File: src/components/WalletProviderModal/WalletCard.js (hunk 1 above)
Comment (OGBoogieTomb) [inline src/components/WalletProviderModal/WalletCard.js:5]: Also consider using `<Button>` from [material ui](https://material-ui.com/components/buttons/) instead.

**Comment by al5ina5** [comment]
Hi, just pushed fixes as requested.

**Review by OGBoogieTomb — APPROVED** [review]
Looks good, thanks for the contribution.

**Comment by al5ina5** [comment]
Thanks a-lot Mr. Tomb! I am really glad I could help.

My FTM address: `0x75066c400313276b9cAb1DE56F3134BAF8B974E5`

Looking forward to the next bounty! 🎉🥳

**Pull request merged**